src/apps/experimental/routes/routes.tsx

```diff
@@ -34,7 +34,7 @@ export const EXPERIMENTAL_APP_ROUTES: RouteObject[] = [
             },
 
             /* Public routes */
-            { index: true, loader: () => redirect('/home.html') },
+            { index: true, element: <Navigate to='/home.html' /> },
```

Comment:
Don't we need `replace` here?

Reply:
By default the `redirect` function doesn't replace. Do you want me to still replace it?

Reply:
Doesn't `redirect` do this via `302` code?

Reply:
Nope, and to confirm it I've also tried it in one of my react projects

Reply:
Hmm, looks like you're right.
Anyway, I would add `replace` to remove redundant history entry (of course, if it will work on Tizen):
| no `replace` | `replace` |
|---|---|
|![history1](https://github.com/user-attachments/assets/924cf780-b3fc-4e26-82d1-f139ac1975d2)|![history2](https://github.com/user-attachments/assets/9a028382-0caf-4deb-aaa5-9c155ca82dc4)|

Reply:
```suggestion
            { index: true, element: <Navigate replace to='/home.html' /> },
```

Reply:
Done on both the files, and also on the PR for release